x-pack/plugins/security_solution/server/endpoint/routes/resolver/utils/pagination.ts

```diff
@@ -61,7 +61,7 @@ export class PaginationBuilder {
     const lastResult = results[results.length - 1];
     const cursor = {
       timestamp: lastResult['@timestamp'],
-      eventID: eventId(lastResult),
+      eventID: String(eventId(lastResult)),
```

Comment:
the behavior here is changed. Before, if the event was a legacy event and the id was `undefined` or `0`this would be `''`. Now it will be `'0'` or `'undefined'`. I'm not sure that either behavior is correct.

Reply:
Thanks for point this out. I think we'll want to check if it is `undefined` and set it to an empty string here. If `0` is returned we'll likely want to keep it as `0` because I think that's a valid value for the legacy events.